transformer/transformer_test.go

```diff
@@ -23,7 +23,7 @@ func TestTransform(t *testing.T) {
 			DatasetID: "",
 		},
 	}
-	expectedESDocument2 := models.ESSourceDocument{
+	expectedItem2 := models.Item{
 		DataType:        "anyDataType2",
 		CDID:            "",
 		DatasetID:       "",
```

Comment:
I would suggest testing that the new transform logic works as expected. This item model can be extended with:
```
		PopulationType: "PopLbl2",
		Dimensions: []models.ESDimensions{
			{Name: "Dim2", Label: "Lbl2", RawLabel: "RawLbl2"},
		},
```

And `esDocument2` in `PrepareESMockResponse` func could be extended with:
```
		PopulationType:  models.ESPopulationType{Name: "Pop2", Label: "PopLbl2"},
		Dimensions: []models.ESDimensions{
			{Name: "Dim2", Label: "Lbl2", RawLabel: "RawLbl2"},
		},
```

Reply:
done